src/Transports/MassTransit.Azure.ServiceBus.Core/AzureServiceBusTransport/Contexts/AzureServiceBusSendContext.cs

```diff
@@ -45,6 +45,8 @@ public AzureServiceBusSendContext(T message, CancellationToken cancellationToken
 
         public DateTime? ScheduledEnqueueTimeUtc { get; set; }
 
+        public string Label { get; set; }
```

Comment:
Needs to be added as a property below (see other comment).

Reply:
I have added the requested changes.